BraveRewardsUI/Publisher/PublisherView.swift

```diff
@@ -69,20 +72,38 @@ class PublisherView: UIStackView {
     $0.spacing = 4.0
   }
   
-  // For containing verificationSymbolImageView and verifiedLabel
+  // For containing verificationSymbolImageView and verifiedCheckAgainStackView
   private let verifiedLabelStackView = UIStackView().then {
     $0.spacing = 4.0
   }
+  
   // ✓ or ?
   private let verificationSymbolImageView = UIImageView().then {
+    $0.contentMode = .scaleAspectFit
     $0.setContentHuggingPriority(.required, for: .horizontal)
   }
+  
+  // For containing verifiedLabel and checkAgainButton
+  private let verifiedCheckAgainStackView = UIStackView().then {
+    $0.spacing = 4.0
+  }
+  
   // "Brave Verified Publisher" / "Not yet verified"
   private let verifiedLabel = UILabel().then {
     $0.textColor = UX.verifiedStatusColor
     $0.font = .systemFont(ofSize: 12.0)
     $0.adjustsFontSizeToFitWidth = true
   }
+  
+  // "Brave Check Again" button
```

Comment:
This comment can be removed, it doesn't say anything useful, the button name below is good enoguh